documentation/src/docs/asciidoc/user-guide/writing-tests.adoc

```diff
@@ -2930,8 +2930,15 @@ parallel with each other but not while any other test that requires `READ_WRITE`
 to the same shared resource is running.
 
 [source,java]
+.Declaring shared resources with `{ResourceLock}` annotation
 ----
-include::{testDir}/example/SharedResourcesDemo.java[tags=user_guide]
+include::{testDir}/example/SharedResourcesWithResourceLockAnnotationDemo.java[tags=user_guide]
+----
+
+[source,java]
+.Declaring shared resources provider with `{ResourceLocksFrom}` annotation
+----
+include::{testDir}/example/SharedResourcesWithResourceLocksFromAnnotationDemo.java[tags=user_guide]
```

Comment:
We should also mention here (and in the annotations' Javadoc) that "static" and "dynamic" locks are combined.

Reply:
Updated. 
Mentioned in Javadoc for `@ResourceLock` and `ResourceLocksProvider`.